.github/stale.yml

```diff
@@ -15,9 +15,29 @@ staleLabel: stale
 # Comment to post when marking an issue as stale. Leave as false
 # to disable.
 markComment: >
-  This issue has been automatically marked as stale because it has not had any
-  activity in the past 30 days. It will be closed in 7 days if no further
-  activity occurs. Thank you for your contributions.
+  Hi! This issue has been automatically marked as stale because it has not had any
+  activity in the past 30 days.
+
+  We use a stalebot among other tools to help manage the state of issues in this project.
+  A stalebot can be very useful in closing issues in a number of cases; the most common
+  is closing issues or PR's where the original reporter has not responded.
+
+  Stalebots are also emotionless and cruel and can close issues which are still very relevant.
+
+  If this issue is important to you, please comment to keep it open.
+
+  **More importantly, please add a thumbs-up to the original issue entry.**
```

Comment:
_More_ importantly? Would we prefer a closed issue with many updoots?

Reply:
when writing this my thought was an issue might not remain open, we might close it with a `revivable` label, or it may just get closed by the stalebot so I wanted to make sure folks were sure to add their support via a thumbs up if the issue was important to them regardless of it staying open or closed